covid19_sfbayarea/data/san_francisco.py

```diff
@@ -168,67 +173,65 @@ def get_age_table(session : SocrataApi, resource_ids: Dict[str, str]) -> List[Di
 def get_gender_table(session : SocrataApi, resource_ids: Dict[str, str]) -> Dict:
     """Get cases by gender"""
 
-   # Dict of source_label:target_label for re-keying.
+    # Dict of source_label:target_label for re-keying.
+    # Note: non cis genders not currently reported
     resource_id = resource_ids['gender']
-    GENDER_KEYS = {"Female": "female", "Male": "male", "Unknown": "unknown"}
-    params = {'$select': 'gender, MAX(cumulative_confirmed_cases) as cumul_cases',
-            '$group': 'gender' }
-    data = session.resource(resource_id, params=params)
-
-    # check that the set of genders present in data is equal to the genders we expect
-    data_genders  = set( [entry['gender'] for entry in data])
-    if GENDER_KEYS.keys() != data_genders:
-        raise FormatError(
-            f"Did not get expected Gender categories. Data received: {data}")
-
-    # re-key
-    gender_table = {GENDER_KEYS[entry["gender"]]: int(
-        entry["cumul_cases"]) for entry in data}
-
-    return gender_table
+    GENDER_KEYS = {"Female": "female", "Male": "male", "Unknown": "unknown", "Trans Female": "female", "Trans Male": "male"}
+    # find the latest date of data collection
+    params = {'$select': 'max(specimen_collection_date) as date'}
+    latest_date = session.resource(resource_id, params=params)[0]
```

Comment:
Same here as above about making this the date string instead of a dict:

```suggestion
    latest_date = session.resource(resource_id, params=params)[0]['date']
```